skyportal/tests/api/test_photometry.py

```diff
@@ -478,13 +478,21 @@ def test_retrieve_photometry_error_group_membership_posted_by_other(
         },
         token=upload_data_token,
     )
+    print(
```

Comment:
Are these debug calls, or useful in the logs?

Reply:
Probably useful in the logs?  But I'll comment them out since we do not need print statements in the test suite as a matter of practice.

Reply:
I don't think it's a big deal to have them there, as long as they provide generally useful information---go with whatever you think is best.